Graphics/Implicit/Primitives.hs-boot

```diff
@@ -0,0 +1,31 @@
+{-# LANGUAGE FlexibleInstances #-}
```

Comment:
... What is this file?

Reply:
Contrary to popular believe, GHC actually supports header files. This is one. Usually they're unnecessary because our modules don't form a dependency loop, but here we do.

```
Graphics.Implicit.Primitives >
Graphics.Implicit.ObjectUtil >
Graphics.Implicit.ObjectUtil.GetImplicit3 >
Graphics.Implicit.ObjectUtil.GetImplicitShared >
Graphics.Implicit.Primitives
```

and similarly for GetBox.

When importing it in `GetImplicitShared`, we import it via ` import {-# SOURCE #-} Graphics.Implicit.Primitives` with the `SOURCE` pragma, rather than the full module.